### ISSUE TITLE
Latin text not rendered in OCRed text

### ISSUE THREAD

**Issue by mirjam-amsterdam** [urieli/jochre#32]
https://ocr.yiddishbookcenter.org/contents?doc=nybc202767#page24

Latin text within the Yiddish text is not rendered, but there also is no placeholder indicating that some text is missing and that the reader should go to the original scan (before preparing an e-book, or before quoting etc.)

![latin text missing 2](https://user-images.githubusercontent.com/44893880/58914428-bcee1900-871e-11e9-81d2-970a4e5084b3.JPG)
![latin text missing 1](https://user-images.githubusercontent.com/44893880/58914429-bd86af80-871e-11e9-91c1-deba56d80e05.JPG)

**Comment by urieli** [comment]
Yes, I made the mistake in the early analyses to configure a "junk setting", which ignores text if the confidence score is too low. This means certain passages (typically other alphabets) are simply skipped. In the newer analyses this should no longer be the case.
However, I'd rather wait for the new version of Jochre to fix this, as this version should be able to handle multiple alphabets.

**Comment by mirjam-amsterdam** [comment]
Stumbled over a misreading: when searching for מאַנש I get a result that actually is in Latin letters Wien !
Please, do make Latin letters searchable and show them as Latin letters in the text. And don't treat me with false results when I am looking for Mansch...
<img width="846" alt="Wien" src="https://user-images.githubusercontent.com/44893880/187141779-27cd01a2-2fda-47de-9f65-52cf9ace78ce.PNG">
<img width="833" alt="source of Mansch - Wien" src="https://user-images.githubusercontent.com/44893880/187141954-05d5b3e1-b0e0-41dd-a2ef-b6ceb40061d9.PNG">
